app/controllers/api/reports_controller.rb

```diff
@@ -40,6 +40,132 @@ def cat3
                                    end_date, provider_filter), content_type: "attachment/xml"
     end
 
+    api :GET, "/reports/patients" #/:id/:sub_id/:effective_date/:provider_id/:patient_type"
+    param :id, String, :desc => "Measure ID", :required => true
+    param :sub_id, String, :desc => "Measure sub ID", :required => false
+    param :effective_date, String, :desc => 'Time in seconds since the epoch for the end date of the reporting period', :required => true
+    param :provider_id, String, :desc => 'Provider ID for filtering quality report', :required => true
+    param :patient_type, String, :desc => 'Outlier, Numerator, Denominator', :required => true
+    description <<-CDESC
+      This action will generate an Excel spreadsheet of relevant QRDA Category I Document based on the category of patients selected. 
+    CDESC
+    def patients
+      type = params[:patient_type]        
+      qr = QME::QualityReport.where(:effective_date => params[:effective_date].to_i, :measure_id => params[:id], :sub_id => params[:sub_id], "filters.providers" => params[:provider_id])
+      
+      authorize! :read, Provider.find(params[:provider_id])
+
+      records = (qr.count > 0) ? qr.first.patient_results : []
+   
+      book = Spreadsheet::Workbook.new
+      sheet = book.create_worksheet
+      format = Spreadsheet::Format.new :weight => :bold		  
+      
+      measure = HealthDataStandards::CQM::Measure.where(id: params[:id]).first
+      
+      eff = Time.at(params[:effective_date].to_i)
+      end_date = eff.strftime("%D")
+      start_date = eff.month.to_s + "/" + eff.day.to_s + "/" + (eff.year-1).to_s
+      # report info
+      sheet.row(0).push "Measure ID: ", "NQF" + measure.nqf_id + ", " + measure.cms_id
+      sheet.row(1).push "Name: ", measure.name
+      sheet.row(2).push "Description: ", measure.description
+      sheet.row(3).push "Reporting Period: ", start_date + " - " + end_date
+      sheet.row(4).push "Group: ", patient_type(type)
+      # table headers
+      sheet.row(6).push 'MRN', 'First Name', 'Last Name', 'Gender', 'Birthdate'
+      sheet.row(6).default_format = format
+      row = 7;
+      (0..4).each do |i| 
+        sheet.row(i).set_format(0, format) 
+      end
+      
+      records.each do |record|
+        value = record.value
+        authorize! :read, Record.find_by(medical_record_number: value[:medical_record_id])
+        if value["#{type}"] == 1
+          sheet.row(row).push value[:medical_record_id], value[:first], value[:last], value[:gender], Time.at(value[:birthdate]).strftime("%D")
+          row +=1
+        end
+      end
+
+      today = Time.now.strftime("%D")  
+      filename = "patients_" + measure.cms_id + "_" + patient_type(type) + "_" + "#{today}" + ".xls"
+      data = StringIO.new '';
+      book.write data;
+      send_data(data.string, {
+        :disposition => 'attachment',
+        :encoding => 'utf8',
+        :stream => false,
+        :type => 'application/excel',
```

Comment:
Consider changing to "application/vnd.ms-excel", see: http://stackoverflow.com/questions/974079/setting-mime-type-for-excel-document